cliquet/permission/postgresql/__init__.py

```diff
@@ -163,6 +163,30 @@ def object_permission_authorized_principals(self, object_id, permission,
             results = cursor.fetchall()
         return set([r['principal'] for r in results])
 
+    def principals_accessible_objects(self, principals, permission,
+                                      object_id_match=None):
+        if object_id_match is None:
+            object_id_condition = 'true'
+        else:
+            object_id_match = object_id_match.replace('*', '%%')
+            object_id_condition = "object_id LIKE '%s'" % object_id_match
```

Comment:
In order to avoid SQL injections, we usually prefer to build the SQL apart from the values we will pass during the query. and then execute the request with `cursor.execute(query, placeholders)`.

I believe it would be saner to actually append text to the query and to the placeholders rather than building everything like this.